lib/src/flutterflow_api_client.dart

```diff
@@ -76,6 +76,10 @@ Future<String?> exportCode({
   String? commitHash,
   bool exportAsDebug = false,
 }) async {
+  stdout.write('Downloading code with the FlutterFlow CLI...\n');
```

Comment:
It's more conventional for CLI tools to write logs like these to stderr - see discussions [1](https://unix.stackexchange.com/questions/331611/do-progress-reports-logging-information-belong-on-stderr-or-stdout), [2](https://stackoverflow.com/questions/4919093/should-i-log-messages-to-stderr-or-stdout).

Since we don't write "regular" output it's probably okay to write these logs to stdout, but IMO it's still a little better to write them to stderr.

Reply:
Gotcha. Good suggestion!